lib/posts/20160422_binary_tree.md

```diff
@@ -68,18 +68,17 @@ The first definition leads to the most concise solution:
 
 ```javascript
 function isSymmetric(root) {
-  return isSymmetricHelper(root.left, root.right);
-}
-
-function isSymmetricHelper(t1, t2) {
-  if (t1 == null && t2 == null) {
-    return true;
-  }
-  if (t1 == null || t2 == null) {
-    return false;
+  function isSymmetricHelper(t1, t2) {
+    if(t1 === null && t2 === null) {
```

Comment:
Using `==` by design because `== null` is a convenient shorthand for checking for both null and undefined